extra/3-magic-8-ball.js

```diff
@@ -1,53 +1,70 @@
-/**
-
-  Let's peer into the future using a Magic 8 Ball!
-  https://en.wikipedia.org/wiki/Magic_8-Ball 
-
-  There are a few steps to being able view the future though:
-  * Ask a question
-  * Shake the ball
-  * Get an answer
-  * Decide if it's positive or negative
-
-  The question can be anything, but the answers are fixed,
-  and have different levels of positivity or negativity.
-
-  Below are the possible answers:
-
-  ## Very positive
-    It is certain.
-    It is decidedly so.
-    Without a doubt.
-    Yes - definitely.
-    You may rely on it.
-
-  ## Positive
-    As I see it, yes.
-    Most likely.
-    Outlook good.
-    Yes.
-    Signs point to yes.
-
-  ## Negative
-    Reply hazy, try again.
-    Ask again later.
-    Better not tell you now.
-    Cannot predict now.
-    Concentrate and ask again.
-
-  ## Very negative
-    Don't count on it.
-    My reply is no.
-    My sources say no.
-    Outlook not so good.
-    Very doubtful.
-*/
+// /**
+
+//   Let's peer into the future using a Magic 8 Ball!
+//   https://en.wikipedia.org/wiki/Magic_8-Ball 
+
+//   There are a few steps to being able view the future though:
+//   * Ask a question
+//   * Shake the ball
+//   * Get an answer
+//   * Decide if it's positive or negative
+
+//   The question can be anything, but the answers are fixed,
+//   and have different levels of positivity or negativity.
+
+//   Below are the possible answers:
+
+
+  let veryPositive = [
+
+    "It is certain.",
+    "It is decidedly so.",
+    "Without a doubt.",
+    "Yes - definitely.",
+    "You may rely on it."
+  ]
+
+  let positive = [
+
+    "Positive",
+    "As I see it, yes.",
+    "Most likely.",
+    "Outlook good.",
+    "Yes.",
+    "Signs point to yes."
+  ]
+
+  let negative = [
+
+    "Reply hazy, try again.",
+    "Ask again later.",
+    "Better not tell you now.",
+    "Cannot predict now.",
+    "Concentrate and ask again."
+  ]
+
+  let veryNegative = [
+
+    "Don't count on it.",
+    "My reply is no.",
+    "My sources say no.",
+    "Outlook not so good.",
+    "Very doubtful."
+  ]
+
 
 // This should log "The ball has shaken!"
 // and return the answer.
+
 function shakeBall() {
-  //Write your code in here
-}
+  console.log("The ball has shaken!");
+  // const allAnswers = veryPositive.concat(positive, negative, veryNegative);
+  const allAnswers = ["very positive", "positive", "negative", "very negative"];
+  let answer = allAnswers [Math.floor(Math.random()*allAnswers.length)];
+  // console.log(prediction);
+  return answer;
```

Comment:
```suggestion
const allAnswers = [...veryPositive, ...positive, ...negative, ...veryNegative];
  let answer = allAnswers[Math.floor(Math.random()*allAnswers.length)];
 
  return answer;
```
Check spread operator. Don't forget to remove unnecessary comments.